app/main.py

```diff
@@ -1 +1,22 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command_split = command.split()
+    if len(command_split) == 3 and command_split[0] == "mv":
+        comm, file_name, path = command_split
+
+    if "/" not in path:
+        os.rename(file_name, path)
+        return
```

Comment:
it's better to raise some error in this case so that the user knows if something goes wrong